takwimu/utils/medium_models.py

```diff
@@ -109,13 +109,11 @@ def post_tags(self, tags):
         self._post_tags = tags
 
     def __str__(self, *args, **kwargs):
-        return str(to_dict(self))
+        return self.title
```

Comment:
QQ @phillipahereza, I see we're switching to just `title` instead of full object representation. Any reason?

Reply:
@kilemensi I was using the full object representation for debugging when I was working on this, but `title` is sufficient to represent the object